xml/System.ComponentModel.DataAnnotations/ScaffoldTableAttribute.xml

```diff
@@ -73,11 +74,12 @@ End Class
   
 -   Visual Studio 2008 Service Pack 1 or Visual Developer 2008 Express Edition Service Pack 1.  
```

Comment:
Again, the entire Example section should be commented out as previously.

Reply:
I think the examples might still work. I'm leaving them for now.